app/models/indexing/scheduled_reindex_job.rb

```diff
@@ -7,6 +7,8 @@ def self.perform(reindex_type)
                 MAIN_CLASSES
               when "stats"
                 %w[StatCounter]
+              when "users"
+                %w[User]
```

Comment:
I left User in `MAIN_CLASSES` because `run_world_index_queue` uses these classes to run things in the world reindex queue, which User can still end up in via Indexer `index_from_db`